Update privatelink-service.md (#7538)
magento/devdocs: src/cloud/project/privatelink-service.md

   * Closing connection 0
   ```
   {:.no-copy}

>    

Reviewer: So the following two steps to check whether connection is established, and to check the package flow   are appended to the procedure  "To test the connection to the VPC endpoint service".  

Last step of that procedure is ```Run curl -v telnet://<endpoint-service-dns-url>:<port>/```

followed by success and failure output.

Do I have to check whether connection is established, if the  telnet request succeeds, or only if it fails.

Same thing with the packages flow.

Is checking for "established connection" and checking package flow always required?  Or should there be a L3 (### Troubleshooting heading before these steps?

Author: Well, it's my bad. Actually, the CURL cmd is to check the connectivity and the NETSTAT is to check if the system has an established connection on whichever port needed. I'll change it to port instead of established.

> 1. Run the following command to check connection is established or not:

Reviewer: ```suggestion
1. Run the following command to check whether the connection is established:
```

   ```bash
   netstat -an |grep ESTABLISHED
   ```   
 

> 1. Run the following command to check the packages flow:

Reviewer: what is the "packages flow"?

   ```bash
   tcpdump -i <ethernet interface> -tt -nn port <destination port> and host <source host>

>    ```   

Reviewer: what should the output be?

   
1. Run the following command to verify service is listening on VM or not:

   ```bash

>    netsat -na |grep <port>

Reviewer: Output -- what if it fails, or gives me unexpected output?

   ```      
  

>    Perform a qucik check on internal settings to make sure everything looks good: 

Reviewer: ```suggestion
   Check the following internal settings to ensure that the configuration is valid: 
```
